syntax="proto3";

package protocol;

message SCPlayerInfo
{
  int32 id = 1;
  string name = 2;
}

message SCOnlinePlayerList
{
  repeated SCPlayerInfo player = 1;
}